cursorless-talon/src/marks/literal_mark.py

```diff
@@ -0,0 +1,28 @@
+from talon import Module
+
+from .mark_types import LiteralMark
+
+mod = Module()
+
+
+# FIXME: Replace literal *heh* with a Talon list before we make this public
```

Comment:
\*heh\* is a joke because it's a command literal named literal, right? It makes sense but took a couple of parses

Reply:
Exactly. I wasn't sure if anyone else would get it :D